presentation/src/main/AndroidManifest.xml

```diff
@@ -128,6 +128,7 @@
         <receiver
             android:name=".receiver.MmsReceivedReceiver"
             android:taskAffinity="com.klinker.android.messaging.MMS_RECEIVED" />
+        <receiver android:name=".receiver.DeleteMessagesReceiver" />
```

Comment:
Could these be moved up so that it's directly under `DefaultSmsChangedReceiver` and the receivers are sorted alphabetically